src/sentry/interfaces/stacktrace.py

```diff
@@ -154,7 +154,6 @@ def to_python(cls, data, **kwargs):
             "symbol_addr",
             "trust",
             "vars",
-            "snapshot",
```

Comment:
This was added to reverse stacktraces which we drop the support for.

Added here:
https://github.com/getsentry/sentry/pull/24346/files#diff-4cc5f7f16f745756dc19e69e113a46b7ec27e615c4323e74914e23a09cd83b37R155

Reply:
CC @jjbayer @untitaker